stack.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from linked_list import LinkedList
+
+
+class Stack():
+
+    def __init__(self, iterable=()):
+        self.other = LinkedList()
+        self.other.__init__(iterable)
+
+    def __repr__(self):
+        return self.other.__repr__()
+
+    def push(self, value):
+        """Add a value to the head of the stack.
+
+        args:
+            value: The value to add to the stack
+        """
+        self.other.insert(value)
+
+    def pop(self):
+        """Remove a value from head of stack and return."""
+        val = self.other.pop()
+
+        return val
```

Comment:
This is fine, but the extra empty line is a bit much.  And many would wonder why you used the local variable `val` here.  Why not just

```python
    return self.other.pop()
```